library/pdns_zone.py

```diff
@@ -144,13 +150,11 @@
           api_port={{ api_port }}
 '''
 
-headers = {
```

Comment:
Why are you changing this formatting? I find the single-line version, while terse, doesn't add clarity.